airbyte-integrations/connectors/source-zendesk-sunshine/requirements.txt

```diff
@@ -0,0 +1,4 @@
+-e ../../bases/airbyte-protocol
+-e ../../bases/base-python
```

Comment:
```suggestion
```
and please apply suggestions using GH feature, in such way it is much easier to understand if you accepted suggested code